<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Student</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #F5F5F5;
    font-family: "Poppins", Arial, sans-serif;
}

QLabel#titleLabel {
    color: #084924;
    font-weight: 600;
}

/* Search input field */
QLineEdit#searchLineEdit {
    padding: 10px;
    border: 1px solid #E0E0E0;
    border-radius: 6px;
    background-color: white;
    font-size: 11pt;
}

QLineEdit#searchLineEdit:focus {
    border: 2px solid #084924;
}

/* Bulk button - Orange/Yellow */
QPushButton#bulkButton {
    background-color: #FFA726;
    color: white;
    border: none;
    padding: 10px 20px;
    border-radius: 6px;
    font-weight: 600;
    font-size: 10pt;
}

QPushButton#bulkButton:hover {
    background-color: #FB8C00;
}

/* Search suggestions container */
QWidget#suggestionsContainer {
    background-color: white;
    border: 1px solid #E0E0E0;
    border-radius: 6px;
}

/* Search suggestion items list */
QListWidget#suggestionsList {
    background-color: transparent;
    border: none;
    padding: 0px;
    outline: none;
}

QListWidget#suggestionsList::item {
    padding: 10px;
    border-bottom: 1px solid #F0F0F0;
    color: #333;
}

QListWidget#suggestionsList::item:hover {
    background-color: #F5F5F5;
}

QListWidget#suggestionsList::item:selected {
    background-color: #E8F5E9;
    color: #084924;
}

/* Plus button next to suggestions */
QPushButton.plusButton {
    background-color: transparent;
    border: none;
    color: #4CAF50;
    font-size: 18pt;
    font-weight: bold;
    padding: 5px;
    min-width: 30px;
    max-width: 30px;
}

QPushButton.plusButton:hover {
    background-color: #E8F5E9;
    border-radius: 4px;
}

/* Students table */
QTableWidget#studentsTable {
    background-color: white;
    border: none;
    border-radius: 10px;
    gridline-color: #F0F0F0;
}

QTableWidget#studentsTable::item {
    padding: 12px;
    border: none;
}

QTableWidget#studentsTable::item:selected {
    background-color: #E8F5E9;
    color: #084924;
}

QHeaderView::section {
    background-color: #084924;
    color: white;
    padding: 12px;
    border: none;
    font-weight: 600;
    font-size: 10pt;
}

/* Cancel button - Gray */
QPushButton#cancelButton {
    background-color: white;
    color: #666;
    border: 2px solid #E0E0E0;
    padding: 12px 32px;
    border-radius: 8px;
    font-size: 11pt;
    font-weight: 600;
}

QPushButton#cancelButton:hover {
    background-color: #F5F5F5;
    border-color: #C0C0C0;
}

/* Enroll button - Green */
QPushButton#enrollButton {
    background-color: #084924;
    color: white;
    border: none;
    padding: 12px 32px;
    border-radius: 8px;
    font-size: 11pt;
    font-weight: 600;
}

QPushButton#enrollButton:hover {
    background-color: #0A5C2E;
}

QPushButton:disabled {
    background-color: #E0E0E0;
    color: #999;
}

QScrollBar:vertical {
    border: none;
    background: #F1F1F1;
    width: 8px;
    border-radius: 4px;
}

QScrollBar::handle:vertical {
    background: #C1C1C1;
    border-radius: 4px;
    min-height: 20px;
}

QScrollBar::handle:vertical:hover {
    background: #A0A0A0;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Add Student</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="searchLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLineEdit" name="searchLineEdit">
       <property name="placeholderText">
        <string>2022301</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bulkButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>42</height>
        </size>
       </property>
       <property name="text">
        <string>BULK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="suggestionsContainer" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="suggestionsLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="suggestionsList">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="studentsTable">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>No</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID Number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Full Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Year Level</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="QLabel" name="countLabel">
       <property name="text">
        <string>0 student(s) to enroll</string>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #666; font-size: 10pt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>45</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="enrollButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>45</height>
        </size>
       </property>
       <property name="text">
        <string>Enroll</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>